source/plugin/plugin-meta.rst

```diff
@@ -108,3 +108,7 @@ your plugin metadata. See :doc:`project/gradle` for details.
 
 .. _Ore: https://github.com/SpongePowered/Ore
 .. _JSON: https://en.wikipedia.org/wiki/JSON
+
+.. note::
+    If you're using the Netbeans IDE make sure you've unchecked the _Compile_ _On_ _Save_ option under
```

Comment:
Minor details, but could you change Netbeans to NetBeans with a capital B to match their official capitalization?

Reply:
Thanks for your comment! Update included in 8dc24ab.